static/bidder-params/appnexus.json

```diff
@@ -103,6 +103,14 @@
         "required": [ "w", "h"]
       },
       "description": "Private sizes (ex: [{\"w\": 300, \"h\": 250},{...}]), experimental, may not be supported."
+    },
+    "ext_inv_code" : {
+      "type": "string",
```

Comment:
Looks like we might have mistakenly gotten the two descriptions swapped? Ref: https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/appnexus.md

Reply:
Thank you, Mansi! updated it in [933c26f](https://github.com/prebid/prebid-server/pull/2792/commits/933c26fdafa4b42aa39d7f089bcd286034a091ab)